src/components/MoneyRequestConfirmationList.js

```diff
@@ -119,16 +122,18 @@ function MoneyRequestConfirmationList(props) {
     const [didConfirm, setDidConfirm] = useState(false);
 
     const splitOrRequestOptions = useMemo(() => {
-        const text = translate(props.hasMultipleParticipants ? 'iou.splitAmount' : 'iou.requestAmount', {
-            amount: CurrencyUtils.convertToDisplayString(props.iouAmount, props.iouCurrencyCode),
-        });
+        const text = props.receiptPath
+            ? translate('iou.request')
+            : translate(props.hasMultipleParticipants ? 'iou.splitAmount' : 'iou.requestAmount', {
```

Comment:
I'm surprised the linter isn't complaining about this, but generally we prohibit nested ternaries for the sake of readability. Suggestion:

```js
let text;
if (props.receiptPath) {
    text = translate('iou.request');
} else {
    const translationKey = props.hasMultipleParticipants ? 'iou.splitAmount' : 'iou.requestAmount';
    text = translate(translationKey, {amount: CurrencyUtils.convertToDisplayString(props.iouAmount, props.iouCurrencyCode)});
}
```